public/app/plugins/panel/alertlist/unified-alerting/GroupedView.tsx

```diff
@@ -8,22 +8,27 @@ import { Alert } from 'app/types/unified-alerting';
 import { CombinedRuleWithLocation } from '../../../../types/unified-alerting';
 import { AlertInstances } from '../AlertInstances';
 import { getStyles } from '../UnifiedAlertList';
-import { GroupedRules, UnifiedAlertListOptions } from '../types';
```

Comment:
nit: with the removal of this import we're no longer using the `GroupedRules` type anywhere from what I can tell. 

We should clean that up and remove the type :)